src/index.html

```diff
@@ -3,11 +3,325 @@
+                >
+              </a>
+            </li>
+            <li class="nav__icon nav__icon--bar">
+            </li>
+          </ul>
+
+          <ul class="nav__links menu" id="menu">
+            <li class="menu__logo-wrapper">
+              <a class="nav__logo-link" href="#home">
+                <img 
+                  class="menu__logo" 
+                  src="images/logo.png" 
+                  alt="logo"
+                >
+              </a>
+            </li>
+
+            <li class="menu__x-wrapper">
+              <img 
+                class="menu__x"
+                src="images/bar-x.png" 
+                alt="logo"
+              >
+            </li>
+            <li class="menu__link-wrapper">
+              <a class="menu__link" href="#home">
+                home
+              </a>
+            </li>
+
+            <li class="menu__link-wrapper">
+              <a class="menu__link" href="#about">
+                about_us
+              </a>
+            </li>
+
+            <li class="menu__link-wrapper">
+              <a class="menu__link" href="#gallery"> 
+                gallery
+              </a>
+            </li>
+            
+            <li class="menu__link-wrapper">
+              <a class="menu__link" href="#testimonials">
+                testimonials
+              </a>
+            </li>
+            
+            <li class="menu__link-wrapper">
+              <a class="menu__link" href="#contacts">
+                contacts
+              </a>
+            </li>
+
+            <li class="menu__link-wrapper menu__link-wrapper--call">
+              <a class="menu__link menu__link--call" href="tel: +48 737 895 912">
+                call to order
+              </a>
+            </li>
+          </ul>
+        </nav>
+        <section class="header__music music">
+          <img
+            src="images/sound-vawes.png"
+            alt="waves"
+            class="music__image music__image--desktop"
+          >
+
+          <img
+            src="images/sound-vawes-1.png"
+            alt="waves"
+            class="music__image music__image--mobile"
+          >
+          <h1>
+            The world shades. 
+            <br>
+            Your music shines
+          </h1>
+        </section>
+      </header>
+      <div class="content">
+        <section class="content__recom recom" id="about">
+          <h2 class="recom__heading">Recommended</h2>
+          <div class="recom__speakers">
+            <div class="recom__image-wrapper">
+              <img 
+                class="recom__image" 
+                src="images/speaker1.png" 
+                alt="speaker1"
+              >
+            </div>
+            <h4 class="recom__description ">Bose portable Smart speaker</h4>
+            <h4 class="recom__description ">Smart home</h4>
+            <h4 class="recom__description ">$ 399.00</h4>
+          </div>
+
+          <div class="recom__speakers">
```

Comment:
the point where these images reduced to 1 column should be near to iPad size(744) not desktop size.
as this results in too much of empty space on the sides
<img width="793" alt="image" src="https://user-images.githubusercontent.com/100297341/184590645-880cc21d-7e22-4b2f-884a-0cc904a95c01.png">